src/test/groovy/org/prebid/server/functional/tests/SetUidSpec.groovy

```diff
@@ -31,4 +54,177 @@ class SetUidSpec extends BaseSpec {
         assert response.responseBody ==
                 ResourceUtil.readByteArrayFromClassPath("org/prebid/server/functional/tracking-pixel.png")
     }
+
+    def "PBS setuid should remove expired uids cookie"() {
+        given: "Default SetuidRequest"
+        def request = SetuidRequest.defaultSetuidRequest
```

Comment:
Hm. Our default request doesn't contain a uuid, which is kinda mandatory. Looks like this should be changed.

Reply:
I don't think so,if it's was mandatory then we could get a error when we try request with empty uid

Reply:
There won't be an error, but PBS also won't return any UIDs too.